lib/fs.js

```diff
@@ -520,7 +520,6 @@ function tryStatSync(fd, isUserFd) {
   } finally {
```

Comment:
Hm, when running the benchmark it does not seem to be a significant change, so it would be more churn than anything else.